http-server/src/test/java/org/triplea/server/lobby/game/listing/GameListingTest.java

```diff
@@ -77,16 +76,13 @@ void setup() {
     /** Basic case, no games added, expect none to be returned. */
     @Test
     void getGames() {
-      final Map<GameId, LobbyGame> games = new HashMap<>();
-      games.put(ID_0, lobbyGame0);
-      games.put(new GameId(API_KEY_0, GAME_ID_1), lobbyGame1);
-      games.put(new GameId(API_KEY_1, GAME_ID_2), lobbyGame2);
-
-      when(cache.asMap()).thenReturn(new ConcurrentHashMap<>(games));
+      cache.put(ID_0, lobbyGame0);
+      cache.put(new GameId(API_KEY_0, GAME_ID_1), lobbyGame1);
+      cache.put(new GameId(API_KEY_1, GAME_ID_2), lobbyGame2);
 
       final List<LobbyGameListing> result = gameListing.getGames();
 
-      assertThat(result.size(), is(games.size()));
+      assertThat((long) result.size(), is(cache.size()));
```

Comment:
nit, I think it's perhaps more conventional to have the hamcrest 'actual' value be as "naked" as possible (so the cast would be moved to the expected value). In this case though, we should probably just use the right matcher to achieve that, eg: ```assertThat(result, IsCollectionWithSize.hasSize(cache.size())```